src/app/search/d2-known-values.ts

```diff
@@ -259,6 +259,31 @@ export const energyNamesByEnum: Record<DestinyEnergyType, string> = {
   [DestinyEnergyType.Stasis]: 'stasis',
 };
 
+/**
+ * Maps TierType to tierTypeName in English and vice versa.
+ * The Bungie.net version of this enum is not representative of real game strings.
+ */
+// A manually constructed bi-directional enum,
+// because the `enum` keyword unfortunately returns type `string`.
+export const D2ItemTiers = {
+  Unknown: TierType.Unknown,
+  [TierType.Unknown]: 'Unknown',
```

Comment:
This being bi-directional in the same object looks odd to me but I guess the `as const` makes it not that different from two objects.

Reply:
it's more or less what a (non-const) `enum` keyword transpiles to: an object with every key=>val and val=>key relationship inserted. just making it very explicit here for output typing purposes